Formula/csound.rb

```diff
@@ -79,6 +93,23 @@ def install
     (lib/"python#{python_version}/site-packages/homebrew-csound.pth").write <<~EOS
       import site; site.addsitedir('#{libexec}')
     EOS
+
+    resource("ableton-link").stage buildpath/"ableton-link"
+
+    resource("csound-plugins").stage do
+      mkdir "build" do
+        system "cmake", "..",
+                        "-DABLETON_LINK_HOME=#{buildpath}/ableton-link",
+                        "-DBUILD_ABLETON_LINK_OPCODES=ON",
+                        "-DBUILD_CHUA_OPCODES=OFF",
+                        "-DCSOUNDLIB=CsoundLib64",
+                        "-DCSOUND_INCLUDE_DIR=#{include}/csound",
+                        "-DCS_FRAMEWORK_DEST=#{frameworks}",
+                        "-DUSE_FLTK=OFF",
```

Comment:
Do we need this flag even if we already have an `fltk` dependency? (Or does changing it to `ON` introduce additional ones?)

Reply:
> Do we need this flag even if we already have an `fltk` dependency?

Sort of. Csound’s [FLTK widgets](https://csound.com/docs/manual/ControlFltkIntro.html) are *currently* being built with the rest of Csound, using code in https://github.com/csound/csound, at

https://github.com/Homebrew/homebrew-core/blob/5c3ed83e3af4e6abdae77a0a3dc0e38d5d448766/Formula/csound.rb#L70

What the flag does is prevent FLTK widgets from being built a second time using code in https://github.com/csound/plugins. If we want to build Csound’s FLTK widgets using code in https://github.com/csound/plugins, we can remove that flag, but we’d need to add it in

https://github.com/Homebrew/homebrew-core/blob/5c3ed83e3af4e6abdae77a0a3dc0e38d5d448766/Formula/csound.rb#L58-L67

Either way, if the situation with Csound’s FLTK widgets ever changes, it’ll be caught in the test (`FLrun`):

https://github.com/Homebrew/homebrew-core/blob/5c3ed83e3af4e6abdae77a0a3dc0e38d5d448766/Formula/csound.rb#L97-L100

Reply:
Ok, that's fine then.